docs/en/observability/profiling-no-kernel-version-check.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[profiling-no-kernel-version-check]]
+= Override kernel version check 
+
+The `-no-kernel-version-check` flag, or the `no-kernel-version-check` key in the host-agent configuration file, controls the kernel version compatibility check during the profiling host-agent's startup process. The kernel version compatibility check enforces the minimum kernel version supported, currently version 4.15, and prevents the profiling agent from running on certain kernel versions with known issues. When the `no-kernel-version-check` is set to `true`, the compatibility check is bypassed, allowing host-agent execution to proceed regardless of the kernel version. By default, this option is set to `false`, and the kernel compatibility version check is performed as usual.
```

Comment:
```suggestion
The `-no-kernel-version-check` flag, or the `no-kernel-version-check` key in the host-agent configuration file, controls the kernel version compatibility check during the profiling host-agent's startup process. The kernel version compatibility check enforces the minimum kernel version supported,  and prevents the profiling agent from running on certain kernel versions with known issues. When the `no-kernel-version-check` is set to `true`, the compatibility check is bypassed, allowing host-agent execution to proceed regardless of the kernel version. By default, this option is set to `false`, and the kernel compatibility version check is performed as usual.
```
I would probably delete this to prevent having to update it in the future unless you think it's important.